ddtrace/__init__.py

```diff
@@ -20,16 +21,15 @@
 
 # configure ddtrace logger before other modules log
 configure_ddtrace_logger()  # noqa: E402
-
-from ddtrace.internal import telemetry
-
-
-telemetry.install_excepthook()
-# In order to support 3.12, we start the writer upon initialization.
-# See https://github.com/python/cpython/pull/104826.
-# Telemetry events will only be sent after the `app-started` is queued.
-# This will occur when the agent writer starts.
-telemetry.telemetry_writer.enable()
+if asbool(os.environ.get("DD_INSTRUMENTATION_TELEMETRY_ENABLED", "true")):
```

Comment:
perhaps we can grab `_telemetry_enabled` from `ddtrace.config` instead?

Reply:
Hmmm, I don't think we could? That would be a circular ref on this file, at least I think it would be.

I can test and get back to you though.